### ISSUE TITLE
Stop linking to old epydoc site 

### ISSUE THREAD

**Issue by tristanlatr** [twisted/pydoctor#347]
We are facing a logistical issue of where and how to document our documentation format... 

Linking to epydoc's old site does not make sens anymore I think since we are now maintaining the reference implementation for Epydoc docstring format parsing. And not everything works the same way epydoc was. 

I think the "epytext demo site" should be converted to references documentation just as the epydoc website, but updated for pydoctor. 

For RST,  i still makes sens to link to docutils documenation. The references documentation (the future RST "demo site") would contain only main features +  extra feature pydoctor ads on top of RST like consolidated fields.

**Comment by mthuurne** [comment]
I agree that it would be better to document epytext as part of pydoctor's documentation. Until we have that, a link to the original documentation is still better than nothing.

**Comment by adiroiban** [comment]
Well, the epytext demo, I have created it as a "demo" and not as "documentation".

But I feel find the old epydoc documentation useful.

To close this ticket the demo code should be refactored to serve as self-demonstrating documentation.
